### PR TITLE
Delete author

### PR DESCRIPTION
**Pull request by LucasQuinotto**

### CODE REVIEW

File: API_Products/database/book_db.py [hunk 1]
```diff
@@ -38,6 +39,40 @@ def isbn_exists_db(isbn_to_check: str) -> bool:
         raise Exception(f"Other PyMongo error: {error.args[0]}")
 
 
+<<<<<<< HEAD
```
Comment (bublitzjr) [inline API_Products/database/book_db.py:42]: Favor resolver confltos

**Review by jrbublitz — CHANGES_REQUESTED** [review]

**Review by jrbublitz — APPROVED** [review]
ok

**Review by Thiago-Alexandre — APPROVED** [review]
Ok

**Review by Vergiliogu — CHANGES_REQUESTED** [review]
Dar pull com a develop

File: API_Products/controllers/author_controller.py [hunk 2]
```diff
@@ -19,3 +19,14 @@ def author_read_all_validation():
...
         except Exception as error:
             return dict(status=400, text=error.args[0])
+
+
+def delete_author(dict_values: dict) -> dict:
+    try:
+        if not author_db.exist_author(dict_values):
+            authors = author_db.delete_authors_db(dict_values)
+        else:
+            raise Exception("A autor já está sendo utilizado, não é possível deletá-lo agora.")
```
Comment (Vergiliogu) [inline API_Products/controllers/author_controller.py:29]: A autor?

File: API_Products/database/author_db.py [hunk 3]
```diff
@@ -20,3 +20,26 @@ def read_all_authors_db() -> list:
...
+    author_lastname = dict_values['lastname']
+    author_country = dict_values['country']
+
+    db = get_db()
+    affected_rows = db.author.delete_one({"name": author_name, "lastname": author_lastname, "country": author_country}).deleted_count
+
+    if affected_rows:
+        return "Registro excluído com sucesso!"
+    else:
+        raise Exception("Nenhum editor encontrado!")
```
Comment (Vergiliogu) [inline API_Products/database/author_db.py:36]: Editor?

**Review by Vergiliogu — CHANGES_REQUESTED** [review]

**Review by Vergiliogu — APPROVED** [review]

**Pull request merged**